lesson17/main.js

```diff
@@ -0,0 +1,173 @@
+const jsonUrl = "https://jsondata.okiba.me/v1/json/YbIZe211030061143";
+const slideshowWrap = document.getElementById("js-slideshow");
+const ul = document.getElementById("js-img-list");
+let imgArray = [];
+
+const createElementWithClassName = (element, name) => {
+  const createdElement = document.createElement(element);
+  createdElement.classList.add(name);
+  return createdElement;
+};
+
```

Comment:
I thought this function is very useful! I will use it as a reference.

Reply:
Thank you. I use it registering it in snippets.